src/conn/conn_prefetch.c

```diff
@@ -186,6 +186,11 @@ __wt_conn_prefetch_queue_push(WT_SESSION_IMPL *session, WT_REF *ref)
         WT_ERR(EBUSY);
     }
 
+    /*
+     * On top of indicating the leaf page is now in the prefetch queue, the prefetch flag also
+     * guarantees the corresponding internal page cannot be evicted until prefetch has processed the
+     * leaf page. This flag is check when eviction reviews an internal page for active children.
```

Comment:
`This flag is check` to `This flag is checked`?

Reply:
Thank you! Fixed in fcad6aa6f97f718136b17cdea145a3f50ecfe29a.